test/sqllogictest/transactions.slt

```diff
@@ -627,6 +627,62 @@ SELECT INTERVAL '1 day'
 statement ok
 COMMIT
 
+# Verify that different kinds of INSERT INTO ... statements are correctly
```

Comment:
Thanks for adding the test. I didn't think of trying this with non-constant values!